src/validator.js

```diff
@@ -376,14 +373,21 @@ export default class Validator {
       });
   }
 
-  extractMetadata(_Xpi=Xpi, _console=console,
-                  _MetadataScanner=MetadataScanner) {
+  extractMetadata({ _Xpi=Xpi, _console=console,
+                    _MetadataScanner=MetadataScanner,
+                    _Directory=Directory } = {}) { // jscs:ignore
```

Comment:
What are we ignoring here, no trailing comma? (Can we ignore just the one rule, just curious.)